reboot-stability-check

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -euo pipefail
+start="${start:-"01"}"
+runs="${runs:-"30"}"
+hosts="${1:?"Need 'hosts' to check reboot stability for. Can be multiple"}"
+boot_timeout="${boot_timeout:-"600"}"
+ping_count="${ping_count:-"30"}"
+sleep_time="${sleep_time:-"120"}"
+
+check-one-host() {
+    local host=$1
+    echo -n "run: $run, $host: ping .. "
+    timeout -k 5 "$boot_timeout" sh -c "until ping -c$ping_count $host >/dev/null; do :; done"
+    echo -n "ok, ssh .. "
+    timeout -k 5 "$boot_timeout" sh -c "until nc -z -w 1 $host 22; do :; done"
+    echo -n "ok, uptime/reboot: "
+    echo ssh "$host" "uptime && sudo reboot"
```

Comment:
Is the `echo` still intended here? It looks like a debugging leftover.

Reply:
yes, thank you. Fixed